packages/custom-field/test/validation.test.js

```diff
@@ -16,19 +17,29 @@ describe('validation', () => {
         </vaadin-custom-field>
       `);
       await nextRender();
+      validateSpy = sinon.spy(customField, 'validate');
     });
 
-    it('should check validity on validate', () => {
```

Comment:
**note:** This test doesn't provide much value since custom-field doesn't override the `validate()` method.